stdlib/public/core/ReflectionLegacy.swift

```diff
@@ -269,6 +269,20 @@ internal struct _StructMirror : _Mirror {
   internal var disposition: _MirrorDisposition { return .`struct` }
 }
 
+@_inlineable // FIXME(sil-serialize-all)
```

Comment:
What does `sil-serialize-all` mean?

Reply:
`sil-serialize-all` is a precursor to `@_inlineable` that stdlib used to build with.  Does what it says on the package: all the SIL used to be serialized so the optimizer could work. The FIXME is because somebody needs to go through and evaluate whether all these functions are profitable to inline.